CONTRIBUTING.md

```diff
@@ -53,7 +53,22 @@ about Git.
   - Create pull request
 
 The title of the pull request should be prefixed by the component or area that
-the pull request affects. Examples:
+the pull request affects. Valid areas as:
+
+  - *Consensus* for changes to consensus critical code
```

Comment:
yep - labels can be used by maintainers to correctly categorize the issues/PRs (and contain many other categories such as 'Questions and Help', 'Upstream', etc). This advice is for individual contributors to mark their own PRs with the component or area they're touching.

Looking at the open PRs, there doesn't seem to be overwhelming consensus one way or the other for square brackets, so I've left the examples below as they were.